apps/participants/views.py

```diff
@@ -466,13 +498,9 @@ def remove_participant_team_from_challenge(
     """
     API to remove the participant team from a challenge
 
-    Arguments:
-        request {HttpRequest} -- The request object
-        challenge_pk {[int]} -- Challenge primary key
-        participant_team_pk {[int]} -- Participant team primary key
-
-    Returns:
-        Response Object -- An object containing api response
+    - Arguments:
```

Comment:
@afif-fahim please revert the changes to docstrings

Reply:
@Ram81 reverted the changes. Preview after reverting:
![image](https://user-images.githubusercontent.com/33936462/109393289-f4691400-794a-11eb-9e35-26ce41592ded.png)